content/hardware/08.edu/solution-and-kits/din-celsius/datasheet/datasheet.md

```diff
@@ -9,13 +9,13 @@ author: Paolo Cavagnolo
 
 # Description
 
-The Arduino DIN Celsius offers you a mini temperature laboratory to test your PLC skills, with two independent heater circuits and one temperature sensor placed at the center of the board.
+The Arduino® DIN Celsius offers you a mini temperature laboratory to test your PLC skills, with two independent heater circuits and one temperature sensor placed at the center of the board.
 
 # Target Areas:
 Pro, PLC projects, Education, Industry Ready, Building automation
 
 # Features
-**Note**: This board needs the Arduino Opta® for full functionality.
+**Note**: This board needs the Arduino Opta® WiFi for full functionality.
```

Comment:
```suggestion
**Note**: This board needs the Arduino Opta® for full functionality.
```
Actually this was written like that on purpose since DIN is compatible with all optas

Reply:
Updated @g-castronovo !